docbook/Admin_Guide/en-US/config/defaults.xml

```diff
@@ -39,6 +39,13 @@
 				</para>
 			</listitem>
 		</varlistentry>
+        <varlistentry>
+            <term>$g_timeline_view_threshold</term>
+            <listitem>
+                <para>The default viewing status for the timeline. The default is VIEWER.
```

Comment:
Revise text to something like: "Threshold for viewing timeline information.  Use NOBODY to turn it off.  The default is VIEWER."